api/src/infrastructure/docker.rs

```diff
@@ -24,1325 +24,1328 @@
-                }
-            })
-        );
-    }
-
-    #[test]
-    fn should_create_service_config_from_container_details() {
-        let details = container_details!(
-            "some-random-id".to_string(),
-            Some(String::from("master")),
-            Some(String::from("nginx")),
-            Some(String::from("nginx")),
-            None,
-        );
-
-        let service = Service::try_from(&details).unwrap();
-
-        assert_eq!(service.id(), "some-random-id");
-        assert_eq!(service.app_name(), "master");
-        assert_eq!(service.config().service_name(), "nginx");
-        assert_eq!(
-            &service.config().image().to_string(),
-            "docker.io/library/nginx:latest"
-        );
-    }
-
-    #[test]
-    fn should_not_create_service_config_from_container_details_with_invalid_image_information() {
-        let details = container_details!(
-            "some-random-id".to_string(),
-            Some(String::from("master")),
-            Some(String::from("nginx")),
-            Some(String::from("\n")),
-            None,
-        );
-
-        let error = Service::try_from(&details).unwrap_err();
-        assert_eq!(
-            error,
-            DockerInfrastructureError::UnexpectedImageFormat {
-                img: String::from("\n"),
-                err: String::from("Invalid image: \n")
-            }
-        );
-    }
-
-    #[test]
-    fn should_create_service_config_from_container_details_without_image_information() {
-        let details = container_details!(
-            "some-random-id".to_string(),
-            Some(String::from("master")),
-            Some(String::from("nginx")),
-            None,
-            None,
-        );
-
-        let service = Service::try_from(&details).unwrap();
-
-        assert_eq!(service.id(), "some-random-id");
-        assert_eq!(service.app_name(), "master");
-        assert_eq!(service.config().service_name(), "nginx");
-        assert_eq!(
-            service.image().to_string(),
-            "sha256:9895c9b90b58c9490471b877f6bb6a90e6bdc154da7fbb526a0322ea242fc913"
-        );
-    }
-
-    #[test]
-    fn should_create_service_config_from_container_details_with_replicated_env() {
-        let details = container_details!(
-            "some-random-id".to_string(),
-            Some(String::from("master")),
-            Some(String::from("nginx")),
-            Some(String::from("nginx")),
-            None,
-            String::from(REPLICATED_ENV_LABEL) => serde_json::json!({ "MYSQL_ROOT_PASSWORD": { "value": "example" } }).to_string()
-        );
-
-        let service = Service::try_from(&details).unwrap();
-
-        assert_eq!(
-            service.config().env().unwrap().get(0).unwrap(),
-            &EnvironmentVariable::with_replicated(
-                String::from("MYSQL_ROOT_PASSWORD"),
-                SecUtf8::from("example")
-            )
-        );
-    }
-}
+ use crate::deployment::deployment_unit::DeploymentUnit;
```

Comment:
It seems that this file has been wrongly formatted. Please, double check and revert. I would expect that many changes… Only a few lines should have been adjusted.